### PR TITLE
[stable/elasticsearch] chown initContainer can be disabled

### PR DESCRIPTION
**Pull request by gzenrique**
Adding the possibility to disable chown initContainer in data and master pods. This allows the chart to run in environments where privileged containers are forbidden. 

This has also been implemented in grafana's chart.
https://github.com/helm/charts/blob/cf3e78d8a426b37073cb2be3d3308d913cd8cd51/stable/grafana/templates/deployment.yaml#L46-L55

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by offzale** [comment]
@desaintmartin @rendhalver any thoughts on this? :)

**Comment by offzale** [comment]
@simonswine @icereval :)

File: stable/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.28.5
+version: 1.28.6
```
Comment (desaintmartin) [inline stable/elasticsearch/Chart.yaml]: This is a new feature: we need to bump minor version instead of patch.

**Comment by desaintmartin** [comment]
Thanks!
/ok-to-test
Could you add one or two value sets in the `ci` test directory proving that it does not crash?

File: stable/elasticsearch/Chart.yaml (hunk 1 above)
Comment (gzenrique) [reply stable/elasticsearch/Chart.yaml]: done :)

**Comment by offzale** [comment]
> Could you add one or two value sets in the `ci` test directory proving that it does not crash?
Sure, will do so

**Pull request merged**